server/graphql/v2/mutation/AccountMutations.ts

```diff
@@ -55,6 +56,17 @@ const accountMutations = {
           }
         }
 
+        if (args.key === 'moderation') {
```

Comment:
I understand that you followed the pattern introduced above with the `collectivePage` settings, but the right place to check that should be in [validateSettings](https://github.com/opencollective/opencollective-api/blob/9aaa9a9393c8c7c720614aa08b19249b9ebe776d/server/lib/collectivelib.js#L103).

The main reason is that this function is called from the model, so regardless of where we try to edit the settings (GQLV1, V2, ForestAdmin, etc.) it will always be called.

Reply:
I did add it there and thru logs found that it called the function, but it didn't throw the error when it wasn't valid when called from this mutation for some reason. I can add it back and maybe you can help troubleshoot.

Reply:
I used `forEach` instead of `for x of x` and that was the problem. Fixed now!